applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -4,954 +4,799 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                     Kratos default license: kratos/license.txt
 //
 //  Main authors:    Riccardo Rossi
 //
-#if !defined(KRATOS_TRILINOS_BLOCK_BUILDER_AND_SOLVER )
-#define  KRATOS_TRILINOS_BLOCK_BUILDER_AND_SOLVER
-
+//
+#if !defined(KRATOS_TRILINOS_BLOCK_BUILDER_AND_SOLVER)
+#define KRATOS_TRILINOS_BLOCK_BUILDER_AND_SOLVER
 
 /* System includes */
-#include <set>
+#include <unordered_set>
 
 /* External includes */
-#include "boost/timer.hpp"
-
 
 /* Project includes */
 #include "includes/define.h"
 #include "solving_strategies/builder_and_solvers/builder_and_solver.h"
+#include "utilities/timer.h"
 
-#include "Epetra_MpiComm.h"
-
-//trilinos includes
-#include "Epetra_Map.h"
-#include "Epetra_Vector.h"
+/* Trilinos includes */
+#include "EpetraExt_MultiVectorOut.h"
+#include "EpetraExt_RowMatrixOut.h"
```

Comment:
minor:
I think you can now remove `EpetraExt_RowMatrixOut.h` and `EpetraExt_MultiVectorOut.h`

Reply:
ok

Reply:
Done !